### PR TITLE
CMR-8863: Investigate graphDb operational issues

### PR DESCRIPTION
**Pull request by eudoroolivares2016**
* Related to ticket, insofar as it is one of the scripts used for analysis to compare the collections on CMR to those in graphDb

* Running python graphqlPaging.py will write the concept-ids of all collections that you have access to according to your TOKEN. These will be printed to a local file collectionConceptIdList.txt

* A fairly minimal example for using python to call cmr-graphql and retrieve data

### CODE REVIEW

File: CMR/python/graphql/graphqlPaging.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    }
+  }
+}"""
+
+variables = """{
+  "params":{
+    "limit":1,
+    "cursor":"%s"
+  }
+}"""% (cursor)
```
Comment (jceaser) [inline CMR/python/graphql/graphqlPaging.py:26]: does pylint complain about this older syntax? You use the modern "f" syntax on line 9 but not here. if pylint says nothing then your fine.

File: CMR/python/graphql/graphqlPaging.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import json
```
Comment (jceaser) [inline CMR/python/graphql/graphqlPaging.py:1]: no description of what this file does, normally I would not care, but you don't have a readme file in the same directory, so no one knows what this is about.

File: CMR/python/graphql/graphqlPaging.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+variables = """{
+  "params":{
+    "limit":1,
+    "cursor":"%s"
+  }
+}"""% (cursor)
+
+response = requests.post(url=endpoint, json={"query": query, "variables": variables}, headers=headers)
+first_result = response.json()
```
Comment (jceaser) [inline CMR/python/graphql/graphqlPaging.py:29]: your going to grab the response before you check the status code?

File: CMR/python/graphql/graphqlPaging.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+        cursor = result.get('data', {}).get('collections', {}).get('cursor')
+        # print("Response in json: ", collections)
+        for collection in collections:
+            arr.append(collection.get('conceptId'))
+        page_num = page_num + 1
+  
+    print('total number of pages', page_num)
+    arr.sort()
+    # Write the sorted concept-id list to a local folder
+    with open("collectionConceptIdList.txt", "w") as txt_file:
```
Comment (jceaser) [inline CMR/python/graphql/graphqlPaging.py:58]: so if you send all your debug statements to standard error, and your output to standard out, you can use pipes and not have to manage file stuff in the code, just an idea. You can just print out the arr values when you get them and not manage the array as well.

basically, all your current print lines could go to stand error, and what you want to go to the file should be standard out to simplify this. If you go down this path, the readme should show an example expected command line call.

Comment (jceaser) [reply CMR/python/graphql/graphqlPaging.py:58]: `print('Error message', file=sys.stderr)`

**Review by jceaser — CHANGES_REQUESTED** [review]
First priority, run pylint if you have not

**Comment by eudoroolivares2016** [comment]
It was suggested that this type of script could live on the future `graphql` landing page so I will close this PR

**Pull request closed without merging**